src/components/Triplers/CardTripler.jsx

```diff
@@ -211,9 +211,33 @@ export class CardTripler extends Component {
         <ListItemText
           primary={`Triplees`}
           secondary={`
-            ${tripler.triplees[0]["first_name"]} ${tripler.triplees[0]["last_name"]},
-            ${tripler.triplees[1]["first_name"]} ${tripler.triplees[1]["last_name"]},
-            ${tripler.triplees[2]["first_name"]} ${tripler.triplees[2]["last_name"]} `}
+            ${
```

Comment:
You can use the `?.` operator as a shorthand for this kind of thing:
```
    {tripler.triplees?.[0]?.first_name || ''} {tripler.triplees?.[0]?.last_name || ''}
```

Actually this probably deserves a function
```
    const getTriplerName = (tripler) => (
      (tripler.triplees?.[0]?.first_name || '') + ' ' + (tripler.triplees?.[0]?.last_name || '')
    ).trim();
```

Then you can do
```
    (tripler.triplees || []).map(getTriplerName).filter(x => x).join(', ')
```

Anyway that's all optional, just useful suggestions for the long term.